learningresources/api.py

```diff
@@ -301,3 +302,20 @@ def import_static_assets(course, path):
                 name = join(root, name).replace(path + sep, '', 1)
                 django_file.name = name
                 create_static_asset(course.id, django_file)
+
+
+def update_xanalytics(filename):
+    """
+    Update xanalytics fields for a LearningResource.
+    Args:
+        filename (string): filename of JSON file from xanalytics
+    """
+    with open(filename) as raw:
+        data = json.load(raw)
```

Comment:
you may want to do some exception handling here, or at least in the calling function.  At the very least you should have a Raises section in the doc string that includes IOError and ValueError